common/src/main/kotlin/com.android.declarative.common/DeclarativeFileParser.kt

```diff
@@ -1,4 +1,4 @@
-package com.android.declarative.internal
+package com.android.declarative.common
```

Comment:
package folder name looks strange: instead of com/android/declarative/common - it's com.android.deveclarative.common